### PR TITLE
tests for str_contains

### PR DESCRIPTION
**Pull request by pbearne**
Trac ticket: https://core.trac.wordpress.org/ticket/49652

### CODE REVIEW

**Review by costdev — COMMENTED** [review]
This review suggests some tidying up in docs and tests.

The methods seem to work as [I would have expected](https://3v4l.org/mb7FE#v5.6.0), although more tests are always welcome. Great work @pbearne!

File: src/wp-includes/compat.php [hunk 1]
```diff
@@ -380,6 +380,25 @@ function is_iterable( $var ) {
...
 }
 
+if ( ! function_exists( 'str_contains' ) ) {
+	/**
+	 * Polyfill for array_key_last() function added in PHP 7.3.
+	 *
+	 * Performs a case-sensitive check indicating if needle is contained in haystack.
+	 *
+	 * @ticket 49652
+	 * @ignore
```
Comment (costdev) [inline src/wp-includes/compat.php:390]: - The `@ticket` annotation is not needed.
- The `@ignore` annotation is not needed.

**Review by hellofromtonya — APPROVED** [review]
Great work @pbearne and @costdev 🎉 Ready for commit.

**Comment by hellofromtonya** [comment]
Committed via changeset https://core.trac.wordpress.org/changeset/52039.

**Pull request closed without merging**